src/SprykerEco/Zed/Heidelpay/Business/HeidelpayFacade.php

```diff
@@ -91,6 +138,36 @@ public function debitPayment(OrderTransfer $orderTransfer): void
             ->debit($orderTransfer);
     }
 
+    /**
+     * {@inheritdoc}
+     *
+     * @api
+     *
+     * @param \Generated\Shared\Transfer\OrderTransfer $orderTransfer
+     *
+     */
+    public function finalizePayment(OrderTransfer $orderTransfer)
+    {
+        $this->getFactory()
+            ->createFinalizeTransactionHandler()
+            ->finalize($orderTransfer);
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @api
+     *
+     * @param \Generated\Shared\Transfer\OrderTransfer $orderTransfer
+     *
+     */
+    public function reservationPayment(OrderTransfer $orderTransfer)
```

Comment:
not a verb.

Reply:
Skipped (@alex-galych ).

Reply:
FYI this is violation of our naming convention.